.travis.yml

```diff
@@ -1,57 +1,25 @@
 language: java
 sudo: false
 dist: trusty
-
 before_install:
-  - mkdir -p ~/bin && curl -sSL -o ~/bin/jq https://github.com/stedolan/jq/releases/download/jq-1.5/jq-linux64 && chmod +x ~/bin/jq
-  - export PATH=$PATH:~/bin
-  - jq --version
-
+- mkdir -p ~/bin && curl -sSL -o ~/bin/jq https://github.com/stedolan/jq/releases/download/jq-1.5/jq-linux64
+  && chmod +x ~/bin/jq
+- export PATH=$PATH:~/bin
+- jq --version
+- export DEV_VERSION="-dev-${TRAVIS_BUILD_NUMBER}"
 jdk:
-  - oraclejdk8
-
+- oraclejdk8
 cache:
   directories:
-    - $HOME/.m2
+  - "$HOME/.m2"
   yarn: true
-
 addons:
   firefox: latest
-
 install:
-  - ./.travis_install.sh
-
-branches:
-  only:
-    - master
-
-jobs:
-  include:
-    - name: "Integration Tests - OracleJDK 8"
-      jdk: oraclejdk8
-      script:
-        - ./.travis_test.sh
-      env:
-        - RUN_INTEGRATION_TESTS=true
-    - name: "Unit Tests - OracleJDK 8"
-      jdk: oraclejdk8
-      script:
-        - ./.travis_test.sh
-      env:
-        - RUN_INTEGRATION_TESTS=false
-    - name: "QuickStart - OracleJDK 8"
-      jdk: oraclejdk8
-      script:
-        - ./.travis_quickstart.sh
-    - name: "QuickStart - OracleJDK 11"
-      jdk: oraclejdk11
-      script:
-        - ./.travis_quickstart.sh
-    - name: "QuickStart - OpenJDK 8"
-      jdk: openjdk8
-      script:
-        - ./.travis_quickstart.sh
-    - name: "QuickStart - OpenJDK 11"
-      jdk: openjdk11
-      script:
-        - ./.travis_quickstart.sh
+- "./.travis_install.sh"
+script:
+- "./.travis_nightly_build.sh"
+env:
+  global:
+  - secure: R3NdobUAThkk6BBzXapek6PiuWHWa1ldnJBvXpk2TAi7Lua/Ctgg8EHCSve7+MK4SU3XFJSi1th32+VSy0K/ARfdVcxiVh6EJSL/Nw0LRQSvGk7VjO382SsWCFoJuZBBqjy9DfBgRLdoqEPfulIBC0XBxjJ+4xedRptkZo7+TAAvtB8Y32EA5ve+Z4cdJipLF1tviL5q2rVbkNXucYUYO2XIv5HOzF2Xy3Hw6lsNrHpbes2CLEkoMiLOQ30zfbXZT/5Xd6OhQNmoZJ9gKziT1opiN2AJMPv269K8FFJZTrDZX74/thq9wmeF4X4f3wPA7EbTupooj0i9LPzb8Xev1QKAsxBtHyRm9NGYOXM06c2yKiGsDtT4HZYQGk4m8/89moe3oGnEwvw/JLu5RvLIZzFE2cBhHrcxRAJgHLIr80+kebNg/YstNlDL+rmP0b9NRrGIVw5OO2BT27ZBgsMFmjnujQtCUJiJ5zqyMxygaYmgZha6xcbhFUPyf4DJ3T46etyFovg2EJnhM7HL9ECZHloQZFhEKs88sZXqvt6HKsA77xyL7ONMrwcPtdDVbFUkN/B1R7LLOf6Wq5vK/Y2EB18oXhKF0ngdahT8HKSMTXjuYf5MPybxhf4fdY6stFmIVkd9b633iZDMu+Aj8E1c8BApyDikCmYK2BI3grH49E0=
```

Comment:
Is there a way to hide this? What the standard practice on putting key? 

For instance, can someone copy this key and make some random release to our repo?

Reply:
No, no need to do so. 
```
Please note that encrypted environment variables are not available for pull requests from forks.
```
First, someone else cannot create a branch from this repo. 
Second, this release deployment will be set up to be triggered only when the build is a cron job.
Third, this release deployment will be set up to be triggered only when the build isn't from PR.